CHANGELOG.md

```diff
@@ -7,6 +7,13 @@
  - Remove Position class in favor of new Vector2 extension
  - Remove Box2D as a dependency
 
+## 0.27.0
```

Comment:
technically this should be [next] :p 
we change it on both develop and sync here when we actually release.
but we can leave as in the end it will be the same